WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -3049,6 +3065,51 @@ public void onTrackableEvent(TrackableEvent event, Map<String, String> propertie
         mEditorTracker.trackEditorEvent(event, mEditorFragment.getEditorName(), properties);
     }
 
+    @Override public void onStoryComposerLoadRequested(ArrayList<Object> mediaFiles, String blockId) {
+        LoadStoryFromStoriesPrefsUseCase loadStoryFromStoriesPrefsUseCase = new LoadStoryFromStoriesPrefsUseCase(
```

Comment:
I would make this use case injectable instead of instantiating it here to improve testability. WDYT?

Reply:
Good idea, done in https://github.com/wordpress-mobile/WordPress-Android/commit/0f4d87c25bde9da952d91fa9a5902a189acc0ed2

Reply:
Nice. Thank you 🙏